frontend/src/employee-frontend/components/child-information/assistance-need/decision/api.ts

```diff
@@ -11,6 +11,8 @@ import { UUID } from 'lib-common/types'
 
 import { client } from '../../../../api/client'
 
+import { AssistanceNeedDecisionForm } from './assistance-need-decision-form'
```

Comment:
I think this should be generated from the server-side types. See my later comment 👇